Exam_Application/models/exam.py

```diff
@@ -0,0 +1,208 @@
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
+
+
+class Examdetails(models.Model):
+    _name = "exam.details"
+    _description = "Exam Informations"
+    _rec_name = "seating_option"
+
+    exam_sequence_number = fields.Char(
+        string="Exam sequence",
+        required=True,
+        readonly=True,
+        copy=False,
+        default="New Sequence",
+    )
+
+    # --------
+    # sequence number generate
+    # --------
+    @api.model
+    def create(self, vals):
+        vals["exam_sequence_number"] = self.env["ir.sequence"].next_by_code(
+            "exam.details")
+        return super(Examdetails, self).create(vals)
+
+    # -------------
+    # search_count()-orm method
+    # -------------
+    exam_search_count = fields.Integer(
+        string="No. of students giving the exam(more than 7) :",
+        compute="action_exam_search_count",
+        readonly=True,
+    )
+
+    @api.depends()
+    def action_exam_search_count(self):
+        for record in self:
+            search_count_total_students = self.env["exam.details"].search_count(
+                [("students_number", ">=", 7)]
+            )
+            self.exam_search_count = search_count_total_students
+
+    exam_name = fields.Char(string="Enter exam name")
+
+    exam_start_date = fields.Date(string="Exam Starting Date")
+
+    exam_end_date_time = fields.Datetime(string="Exam ending Date & Time")
+
+    exam_confirmation = fields.Boolean(string="Confirm to hold the exam(True/False)")
+
+    exam_fees = fields.Float(string="Exam holding fee")
+
+    students_number = fields.Integer(string="Enter total students giving this exam")
+    # color_widget = fields.Integer(string="Color Picker")
```

Comment:
Add short string name like Is Exam Confirmed.
You can add help parameter for display long message.